cedar-policy/src/api.rs

```diff
@@ -2489,6 +2739,49 @@ impl Context {
     /// if attributes have the wrong types (e.g., string instead of integer).
     /// Since different Actions have different schemas for `Context`, you also
     /// must specify the `Action` for schema-based parsing.
+    /// ```
+    /// use cedar_policy::{Context, RestrictedExpression, Schema};
+    /// use std::collections::HashMap;
+    /// use std::str::FromStr;
+    /// # use cedar_policy::{Entities, EntityId, EntityTypeName, EntityUid, Request,PolicySet};
+    /// let data = serde_json::json!(
+    /// {
+    ///     "sub": "1234"
+    /// });
+    /// let schema_data =r#"
+    ///     {
+    ///       "": {
+    ///         "entityTypes": {},
+    ///           "actions": {
+    ///             "view": {
+    ///                "appliesTo": {
+    ///                  "principalTypes": [],
+    ///                   "resourceTypes": [],
+    ///                   "context": {
+    ///                     "type": "Record",
+    ///                     "attributes": {
+    ///                       "sub": { "type": "Long" }
+    ///                     }
+    ///                   }
+    ///                 }
+    ///               }
+    ///           }
+    ///       }
+    ///     }"#;
+    /// # // create a request
+    /// # let p_eid = EntityId::from_str("alice").unwrap();
+    /// # let p_name: EntityTypeName = EntityTypeName::from_str("User").unwrap();
+    /// # let principal = EntityUid::from_type_name_and_id(p_name, p_eid);
+    /// # let a_eid = EntityId::from_str("view").unwrap();
+    /// # let a_name: EntityTypeName = EntityTypeName::from_str("Action").unwrap();
+    /// # let action = EntityUid::from_type_name_and_id(a_name, a_eid);
```

Comment:
I think it would be better to not hide the definition of `action` in this case, since it matters for the behavior of `Context::from_json_value`

Reply:
done